src/main/pg/sdcard.h

```diff
@@ -29,6 +29,10 @@ typedef struct sdcardConfig_s {
     uint8_t cardDetectInverted;
     uint8_t dmaIdentifier;
     uint8_t dmaChannel;
+#ifdef USE_SDCARD_SDIO
```

Comment:
Parameter groups are planned to be used as data structure for the exchange of configuration information over MSP. As such, they rely on a given parameter group of the same name and version always having the same format, save potential additions at the end. For this reason, contents of structs used in parameter groups cannot have conditional elements.

Reply:
Sure I will fix all others. For this should I add seperate pg SDIO?

Reply:
I guess that makes sense - this is really a feature that a board either has or doesn't have.